src/server/pkg/storage/chunk/chunk_test.go

```diff
@@ -102,52 +99,52 @@ func BenchmarkRollingHash(b *testing.B) {
-			finalDataRefs = append(finalDataRefs, a.NextDataRef)
-		}
-		return nil
-	}
-	w := chunks.NewWriter(context.Background(), averageBits, f, 0)
-	r1 := chunks.NewReader(context.Background())
-	r1.NextRange(dataRefs1)
-	r2 := chunks.NewReader(context.Background())
-	r2.NextRange(dataRefs2)
-	w.Annotate(&Annotation{
-		NextDataRef: &DataRef{},
-	})
-	mid := r1.Len() / 2
-	require.NoError(t, w.Copy(r1, r1.Len()-mid))
-	require.NoError(t, w.Copy(r1, mid))
-	mid = r2.Len() / 2
-	require.NoError(t, w.Copy(r2, r2.Len()-mid))
-	require.NoError(t, w.Copy(r2, mid))
-	require.NoError(t, w.Close())
-	// Check that the initial data equals the final data.
-	buf := &bytes.Buffer{}
-	finalR := chunks.NewReader(context.Background())
-	finalR.NextRange(finalDataRefs)
-	_, err := io.Copy(buf, finalR)
-	require.NoError(t, err)
-	require.Equal(t, append(seq1, seq2...), buf.Bytes())
-	// Only one extra chunk should get created when connecting the two sets of data.
-	var finalChunkCount int64
-	require.NoError(t, chunks.List(context.Background(), func(_ string) error {
-		finalChunkCount++
-		return nil
-	}))
-	require.Equal(t, initialChunkCount+1, finalChunkCount)
-}
+//func TestCopy(t *testing.T) {
+//	objC, chunks := LocalStorage(t)
+//	defer Cleanup(objC, chunks)
+//	// Write the initial data and count the chunks.
+//	dataRefs1, seq1 := Write(t, chunks, 60, 20)
+//	dataRefs2, seq2 := Write(t, chunks, 60, 20)
+//	var initialChunkCount int64
+//	require.NoError(t, chunks.List(context.Background(), func(_ string) error {
+//		initialChunkCount++
+//		return nil
+//	}))
+//	// Copy data from readers into new writer.
+//	var finalDataRefs []*DataRef
+//	f := func(_ *DataRef, annotations []*Annotation) error {
+//		for _, a := range annotations {
+//			finalDataRefs = append(finalDataRefs, a.NextDataRef)
+//		}
+//		return nil
+//	}
+//	w := chunks.NewWriter(context.Background(), averageBits, f, 0)
+//	r1 := chunks.NewReader(context.Background())
+//	r1.NextRange(dataRefs1)
+//	r2 := chunks.NewReader(context.Background())
+//	r2.NextRange(dataRefs2)
+//	w.Annotate(&Annotation{
+//		NextDataRef: &DataRef{},
+//	})
+//	mid := r1.Len() / 2
+//	require.NoError(t, w.Copy(r1, r1.Len()-mid))
+//	require.NoError(t, w.Copy(r1, mid))
+//	mid = r2.Len() / 2
+//	require.NoError(t, w.Copy(r2, r2.Len()-mid))
+//	require.NoError(t, w.Copy(r2, mid))
+//	require.NoError(t, w.Close())
+//	// Check that the initial data equals the final data.
+//	buf := &bytes.Buffer{}
+//	finalR := chunks.NewReader(context.Background())
+//	finalR.NextRange(finalDataRefs)
+//	_, err := io.Copy(buf, finalR)
+//	require.NoError(t, err)
+//	require.Equal(t, append(seq1, seq2...), buf.Bytes())
+//	// Only one extra chunk should get created when connecting the two sets of data.
+//	var finalChunkCount int64
+//	require.NoError(t, chunks.List(context.Background(), func(_ string) error {
+//		finalChunkCount++
+//		return nil
+//	}))
+//	require.Equal(t, initialChunkCount+1, finalChunkCount)
+//}
```

Comment:
Why not just `t.Skip` these? Commented out tests almost always bitrot and eventually get deleted because they stop compiling.